docs/distributions/debian/installation.md

```diff
@@ -0,0 +1,91 @@
+
+!!! Warning "Pop!_OS"
+    Due to a bug in Pop!_OS installer, the partition sizes shown by it are incorrect during manual partitioning step. As a workaround you may follow the instructions given in this [GitHub issue](https://github.com/elementary/installer/issues/620) in the live ISO environment to fix the installer and then start the installation.
+
+1. Follow the [Pre-installation](https://wiki.t2linux.org/guides/preinstall) steps to prepare your Mac for installation.
+2. Boot into the Live ISO. You should now be in the GRUB boot menu. Select the option which is relevent to you.
+3. Start the installer and install it like normal until you get an option to manually specify partitions.
+4. Find the partition you made for Linux when you were following the Pre-installation steps. MAKE SURE TO SELECT THE RIGHT PARTITION OR ELSE YOUR DATA WILL BE LOST. Delete it to make free space. You'll need to make these partitions:
+
+    1. If you want, you can make seperate partitions for **swap**, `/home`, `/boot` etc as you do in a normal PC.
+
+    2. The partition to be made compulsorily is the one mounted at `/` and formatted to **ext4** or **btrfs**.
+
+    3. If the installer used by your distro is `ubiquity`, which is the one used in Ubuntu, Linux Mint etc., then you can leave EFI Boot alone. If you are using a [separate EFI partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions), then you shall have to separate it out after installation by following [this guide](https://wiki.t2linux.org/guides/windows/#seperate-the-efi-partition-after-linux-is-installed).
+  
+       For other installers, you need to mount `nvme0n1p1`, or your [separate EFI partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions) (whatever case applies to you) at `/boot/efi`. If the installer supports the "boot￼" flag, set it as well for your EFI partition.
+
+5. Continue the rest of the setup.
+6. Once it's finished, you can reboot without your installation media. Hold down Option (⌥) while booting, then select EFI Boot and press enter.
+
+# Adding T2 support
+
+After installation, we need to install a kernel having patches to support the T2 Macs for the internal keyboard, trackpad, touchbar, audio, Wi-Fi etc. to work. In order to do so :-
+
+1. Boot into your new installation.
+2. Connect to the internet using Ethernet/USB tethering/external Wi-Fi adapter.
+3. Add the **t2-ubuntu-repo** apt repo by running :-
+  
+   ```bash
+   curl -s --compressed "https://adityagarg8.github.io/t2-ubuntu-repo/KEY.gpg" | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/t2-ubuntu-repo.gpg >/dev/null
+   sudo curl -s --compressed -o /etc/apt/sources.list.d/t2.list "https://adityagarg8.github.io/t2-ubuntu-repo/t2.list"
+   sudo apt update
+   ```
+  
+4. Now install the kernel upgrade script.
+  
+   If your distro is Ubuntu based, run :-
```

Comment:
maybe consider using dot points for these

e.g. 

1. Now install the kernel upgrade script.
     - Ubuntu
       ```bash
       foo
       ```
     - Debian
       ```bash
       foo
       ```